st2common/st2common/models/db/liveaction.py

```diff
@@ -75,6 +75,7 @@ class LiveActionDB(stormbase.StormFoundationDB):
         default={},
         help_text='Information about the runner which executed this live action (hostname, pid).')
     notify = me.EmbeddedDocumentField(NotificationSchema)
+    log = me.ListField(field=me.DictField())
```

Comment:
We used to have three models a) LiveAction b) Execution c) History. We realized we need only two things: 1. A lightweight model as a control message to track lifetime of action until it begins execution and 2) An actual execution model that contains the data and the context for the execution. Now we have come close to deprecating LiveAction multiple times but have realized cases where we need two. I cannot recollect a specific example now. The rule of thumb we now have is LiveAction is a system internal that should never be exposed to users and execution is a user consumable model.